### DISCUSSION TITLE
Should we have a new logo?

### DISCUSSION THREAD

**Discussion by mishmanners** [mishmanners/GitHub-Like-A-Boss#94] [Q&A]
Thoughts people?

**Comment by GMBermeo** [comment]
@mishmanners yes
![Captura de Tela 2022-07-12 às 12 26 48](https://user-images.githubusercontent.com/40867634/178527128-ea4ada11-b26d-4a89-ae1b-ec866a2837a2.png)

**Reply by yonathanavila** [reply]
Nice picture

**Reply by mishmanners** [reply]
I love this! So good!
